sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -32,6 +32,10 @@ public partial class TableClient
         public virtual Azure.Response Upsert(System.Collections.Generic.IDictionary<string, object> entity, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> UpsertAsync(System.Collections.Generic.IDictionary<string, object> entity, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> UpsertAsync<T>(T entity, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) where T : Azure.Data.Tables.TableEntity, new() { throw null; }
+        public virtual Azure.Response UpsertMerge(System.Collections.Generic.IDictionary<string, object> entity, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Just curious - is this naming going to appear across all languages? Did we decide against using the existing service naming of `InsertOrMerge`?

Reply:
Oh, I didn't realize there was already an existing name! I think that'll be better for consistency with the previous service. I'll make that change.

Reply:
Although looking at the current service method names, the current `Upsert` should be `InsertOrReplace` then. I think I was trying to align to the `Upsert` that we have right now when using `UpsertMerge`.

Reply:
Yeah, `Upsert` was a new name that I introduced as an idea to simplify the need for `InsertOrReplace`. But if we are going to settle on splitting out the methods across languages, then I'd agree that we should err on the side of service consistency.